src/components/Episodes/renderUtils.jsx

```diff
@@ -1,17 +1,18 @@
 import React from "react";
 import Episode from "COMPONENTS/Episode/";
+import UTILS from "SHARED/utils.js";
 
 function renderEpisode(data, index){
 
 	const { 
 		number,
 		title,
 		synopsis,
-		image,
-		video,
+		thumbnail,
+		trailer,
 	} = data;
 
-	const id = `episode__${number}`;
+	const id = UTILS.convertToHyphenatedString(title);
```

Comment:
see my note in `utils` about why we should use `convertToSafeString` here instead.